prioritize.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+import click
+from PIL.Image import Image as PILImage
+import torch
+import numpy as np
+
+from img2vec_pytorch import Img2Vec
+
+from kili.client import Kili
+from utils.constants import (
+    InputType,
+)
+from utils.helpers import (
+    clear_joblib_cache,
+    download_project_images,
+    get_assets,
+    get_project,
+    kili_print,
+    parse_label_types,
+)
+
+
+def embeddings_images(
+    images: list[PILImage],
+) -> np.ndarray:
+    """Get the embeddings of the images using a generic model trained on ImageNet."""
+    img2vec = Img2Vec(cuda=torch.cuda.is_available())
```

Comment:
what's the behavior if cuda is not available?

Reply:
It works with the cpu. For example, it works on the mac

Reply:
ok, I misunderstood 😄